src/components/pages/CurriculumInfo/tabs/UnitsTab/UnitsTab.tsx

```diff
@@ -372,7 +376,7 @@ const UnitsTab: FC<UnitsTabProps> = ({ data }) => {
               </RadioGroup>
             </Box>
             <Box $mr={16} $mb={32}>
-              <Heading tag={"h3"} $font={"heading-7"} $mb={12}>
+              <Heading tag={"h4"} $font={"heading-7"} $mb={12}>
                 Year Group
```

Comment:
@HelenHarlow Should this be `Year group` (i.e sentence case)?

Reply:
Yeah good spot, thats correct

Reply:
✅